pkg/log/status.go

```diff
@@ -257,6 +257,14 @@ func Swarningf(format string, a ...interface{}) string {
 	return fmt.Sprintf(" %s%s%s", yellow(getWarningString()), suffixSpacing, fmt.Sprintf(format, a...))
 }
 
+func Deprecate(what, nextAction string) {
+	yellow := color.New(color.FgYellow).SprintFunc()
```

Comment:
The declaration of yellow is only necessary in the `if !IsJSON()` block (same for the following functions)

Reply:
Yea actually. Infact that would be true for all our logging function. Updating